WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginPrologueFragment.java

```diff
@@ -25,19 +28,32 @@
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.login_signup_screen, container, false);
 
-        view.findViewById(R.id.login_button).setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if (mLoginPrologueListener != null) {
-                    mLoginPrologueListener.showEmailLoginScreen();
-                }
+        MaterialCardView bottomButtonsCard = view.findViewById(R.id.bottom_buttons);
+
+        if (BuildConfig.UNIFIED_LOGIN_AVAILABLE) {
+            bottomButtonsCard.removeAllViews();
+            inflater.inflate(R.layout.login_prologue_bottom_buttons_container_unified, bottomButtonsCard);
+        }
+
+        MaterialButton primaryButton = view.findViewById(R.id.login_button);
```

Comment:
What do you think about changing the IDs of these two buttons to something more generic? I understand `primary` and `secondary` doesn't really work here since the order is switched between the two designs but maybe something about the order? `first_button` and `second_button`? It might make it a little more readable for the future.

Reply:
Good point! Since these IDs had not been updated since the creation of that screen, I thought it might have some not-so-obvious reason for that and decided not to touch them. Upon closer inspection, I couldn't find any good reason not to change them, so I'll just go with your suggestion 🙂

Thanks for reviewing this! 🙇‍♂️

Reply:
> maybe something about the order? `first_button` and `second_button`?

Oh, I just realized that this could make things look weird on the landscape layout, since the order kind of changes there (the top button becomes the right button and the bottom button becomes the left one), meaning we would have the `first_button` on the right and the `second_button` on the left. This might still be a better alternative though. Let me know what you think.

Reply:
After talking to @mattmiklic about this, we reached the conclusion that it's fine to have the natural order of the buttons on landscape for the new layouts, instead of having them reversed. So I'm renaming the buttons to what you've sugggested and updating the landscape layout for the new layouts, but I'm keeping the old ones as they were. That means we will still have one layout where the `second_button` is coming first, but that will be solved sooner than later, when we get rid of the feature flag and stop maintaining the old flow.

Reply:
Here's a visual before and after:

**Before**
| Portrait | Landscape |
|---|---|
| ![old-portrait](https://user-images.githubusercontent.com/830056/80739034-3982a680-8aec-11ea-8a5b-f03fa101e57b.jpeg) | ![old-landscape](https://user-images.githubusercontent.com/830056/80739027-37204c80-8aec-11ea-9e10-623b129f18e9.jpeg) |

**After**
| Portrait | Landscape |
|---|---|
| ![new-portrait](https://user-images.githubusercontent.com/830056/80739041-3b4c6a00-8aec-11ea-9941-86bf4da03d78.jpeg) | ![new-landscape](https://user-images.githubusercontent.com/830056/80739039-3a1b3d00-8aec-11ea-893e-734b4826b3a8.jpeg) |

Reply:
Looks good to me!